metricbeat/module/kubernetes/state_horizontalpodautoscaler/_meta/test/ksm.v2.10.0.plain.expected

```diff
@@ -0,0 +1 @@
+null
```

Comment:
I believe you should also add the HPA metrics in `ksm.v2.10.0.plain` so that this file is not null. In a later PR we can remove the support for ksm 2.10.0

Reply:
Correct to allign with what we have in other metricsets

Reply:
added in [47ae556](https://github.com/elastic/beats/pull/42392/commits/47ae556a3455f8326cb101db317b9a24f3d79d22)